content/blog/example/en.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Take a cue to supercharge your configurations
```

Comment:
Maybe also include Example in the title so it's clear you've landed on the example blog.

Reply:
yes